printer_mods/README.md

```diff
@@ -25,6 +25,7 @@ like so:
 
 | Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW |
 | --- | --- | --- | --- | --- | --- | --- |
+|LoCoCNC| [MGN7H Shims](./LoCoCNC/) | Shims for making bearing cartridges coplanar |:heavy_check_mark:|:x:|:x:|:x:|
```

Comment:
Please move this down the list so it's in alphabetical order, looks like your line should be after `littleferris`.